tests/com/tony/server/AuthenticatorTest.java

```diff
@@ -0,0 +1,74 @@
+package com.tony.server;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class AuthenticatorTest {
+    @Test
+    public void testAddToAuthenticationList() throws Exception {
```

Comment:
I can see this is consistent with most, if not all, of your other tests, but is there a reason you choose to prepend your test method name with test?